src/parser/grammar.pegjs

```diff
@@ -415,9 +415,493 @@ MapsTo
 
 // ## TPTP syntax
 
+tff = WS "tff(" WS n:name "," WS r:formulaRole "," WS  f:tffFormula WS ")." WS
+        { return {name:n, type:r, formula:f }; }
+
+name
+    = i:Identifier
+        { return i; }
+
+formulaRole
+    = "axiom"
+        { return "axiom"; }
+    / "type"
+        { return "type"; }
+
+tffWS
+    = ('%' WS [\u0020-\u007E]* [\t\n\r\v\f]+)+
+        {return "comment"}
+
+    / WS
+
+
+tffFormula
+    = a:tffAtomTyping t:tffWS
+        { return a }
+    / l:tffLogicFormula tffWS
+        { return l }
+    / tffWS s:tffSubtype tffWS
+        { return s }
+    / tffWS t:tfxSequent tffWS
+        { return t}
+    / t:tffWS
+        {return t}
+
+tffLogicFormula
+    = b:tffBinaryFormula
+          { return b }
+    / un:tffUnaryFormula
+        { return un }
+    / d:tffDefinedInfix
+         { return d}
+    /  u:tffUnitaryFormula
+        { return u}
+
+tffUnitaryFormula
+    = q:tffQuantifiedFormula
+                    { return q}
+    /  "(" WS l:tffLogicFormula ")" WS
+                         { return l}
+    /a:tffAtomicFormula
+                   { return a }
+    / u:tfxUnitaryFormula
+              { return u }
+
+tfxUnitaryFormula
+    =  v:variable WS
+    {return v}
+
+tffQuantifiedFormula
+    =  q:fofQuantifier "[" WS v:tffVariableList "]" WS ":" WS u:tffUnitFormula
+        { return q(v.variable, v.type, u, ee) ;}
```

Comment:
Ak upravíte `tffVariableList` tak, že vždy vráti pole, potom tu to pole treba zredukovať na viacero vnorených kvantifikátorov pomocou [reduceRight()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduceRight) pravdepodobne takto:
```javascript
q:fofQuantifier "[" WS vars:tffVariableList "]" WS ":" WS u:tffUnitFormula
         { return vars.reduceRight( (subf, {variable, type}) => q(variable, type, subf, ee), u) ;}
```
Pre jednu premennú sa na výsledku nič nezmení. Pre viac premenných by mali vzniknúť vnorené kvantifikátory.